src/App.tsx

```diff
@@ -10,9 +32,17 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="/home" element={<Navigate to="/" replace />} />
+            <Route path="/people" element={<PeoplePage />}>
+              <Route path="/people/:slug" element={<PeoplePage />} />
+            </Route>
```

Comment:
The nested Route inside '/people' is incorrect. You should not nest a Route inside another Route unless you are using an Outlet for nested routes. Consider removing the nested Route or using an Outlet if nested routing is intended.